android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -265,6 +265,28 @@ public void isAirPlaneMode(Promise p) {
     p.resolve(isAirPlaneMode);
   }
 
+  @ReactMethod
+  public void isAutoDateAndTime(Promise p) {
+    boolean isAutoDateAndTime;
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.JELLY_BEAN_MR1) {
+      isAutoDateAndTime = Settings.System.getInt(this.reactContext.getContentResolver(),Settings.System.AUTO_TIME, 0) != 0;
+    } else {
+      isAutoDateAndTime = Settings.System.getInt(this.reactContext.getContentResolver(),Settings.Global.AUTO_TIME, 0) != 0;
+    }
+    p.resolve(isAutoDateAndTime);
```

Comment:
Could you use the if to just pickup `Settings.System.AUTO_TIME` or `Settings.Global.AUTO_TIME` to make the code more readable?

```
String name;

if (Build.VERSION.SDK_INT < Build.VERSION_CODES.JELLY_BEAN_MR1) {
    name = Settings.System.AUTO_TIME;
} else {
    name = Settings.Global.AUTO_TIME;
}

p.resolve(Settings.System.getInt(this.reactContext.getContentResolver(), name, 0) != 0);
```